src/sentry/snuba/metrics/extraction.py

```diff
@@ -49,6 +50,41 @@
 
 logger = logging.getLogger(__name__)
 
+
+# This helps us control the different spec versions
+# in order to migrate customers from invalid specs
+class SpecVersion(NamedTuple):
+    version: int
+    flags: Sequence[str] = []
+
+    def has_flag(self, flag: str) -> bool:
+        return flag in self.flags
+
+
+# This is the lower spec version which we collect metrics for.
+# Once we're ready to abandon a version bump this value
+MIN_VERSION = 0
```

Comment:
We need to double-check that widgets are using the spec with the env before bumping this.